cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -223,10 +225,13 @@ function (google_cloud_cpp_enable_cleanup)
 
     list(REMOVE_DUPLICATES GOOGLE_CLOUD_CPP_ENABLE)
 
-    set(GOOGLE_CLOUD_CPP_ENABLE_GRPC ON)
-    if ("${GOOGLE_CLOUD_CPP_ENABLE}" STREQUAL "storage")
-        set(GOOGLE_CLOUD_CPP_ENABLE_GRPC OFF)
-    endif ()
+    set(GOOGLE_CLOUD_CPP_ENABLE_GRPC OFF)
+    foreach (feature ${GOOGLE_CLOUD_CPP_ENABLE})
```

Comment:
It seems we could do something like:

```cmake
set(grpc_features ${GOOGLE_CLOUD_CPP_ENABLE})
list(REMOVE_ITEM grpc_features ${GOOGLE_CLOUD_CPP_REST_ONLY_FEATURES})
set(GOOGLE_CLOUD_CPP_ENABLE_GRPC OFF)
if (grpc_features)
  set(GOOGLE_CLOUD_CPP_ENABLE_GRPC ON)
endif ()
```

Needs some testing though.

Reply:
Apparently not tested well.